docs/extensions/filters/writing_custom_filters.md

```diff
@@ -133,11 +133,13 @@ To extend Quilkin's code with our own custom filter, we need to do the following
    #         unimplemented!()
    #     }
    # }
-   use quilkin::{filters::DynFilterFactory, runner::run};
+   use quilkin::{config::Config, filters::DynFilterFactory, runner::run};
 
    #[tokio::main]
-   async fn main() {
-       run(vec![Box::new(GreetFilterFactory) as DynFilterFactory]).await.unwrap();
+   async fn main() -> Result<(), Box<dyn std::error::Error>> {
+       let log = quilkin::proxy::logger();
+       let config = Config::find(&log, None).map(std::sync::Arc::new)?;
+       run(log, config, vec![Box::new(GreetFilterFactory) as DynFilterFactory]).await
```

Comment:
I think the example should use the shorter version of `run` otherwise we'll need to explain the extra arguments which doesn't seem too useful to the tutorial